### PR TITLE
 add instructions to verify the tarballs using cosign #406 

### PR DESCRIPTION
**Pull request by rod4n4m1**
Updated README with the instructions for verifying a release tarball. 
Issue #406

### CODE REVIEW

**Review by kranurag7 — CHANGES_REQUESTED** [review]
Thanks for the PR @rod4n4m1, really appreciate it. 

Let's use exported variables so that it's easier to replicate on a system. 
We can use something like `export KARMOR_VERSION=v1.1.0` and then we use the same while invoking the commands in the upcoming steps. 

Also, given the commands are long, we can wrap it under a toggle block in markdown. 

<details><summary>Details</summary>
<p>

```bash
# steps to verify the binary
```


</p>
</details> 

I would also move the verification docs after the installation docs.

**Comment by rod4n4m1** [comment]
@kranurag7 Thanks, just changed accordingly to above instructions.

File: README.md [hunk 1]
```diff
@@ -7,23 +7,61 @@
...
+export KARMOR_VERSION="v1.1.0"
+```
+
+- Download release tarball, certificate, and signature files
+
+<details>
+  <summary>Details</summary>
+
+```shell
+curl -LO https://github.com/kubearmor/kubearmor-client/releases/download/${KARMOR_VERSION}/karmor_1.1.0_linux_amd64.tar.gz
```
Comment (kranurag7) [inline README.md:42]:
```suggestion
curl -LO https://github.com/kubearmor/kubearmor-client/releases/download/${KARMOR_VERSION}/karmor_${KARMOR_VERSION#v}_linux_amd64.tar.gz
```

We can use version in the name as well. Or we can export `KARMOR_VERSION=1.1.0` and use `v${KARMOR_VERSION}`

You can choose either of them.

Comment (rod4n4m1) [reply README.md]: Thanks, this is niuce

**Comment by rod4n4m1** [comment]
@kranurag7 Done, please check. Thanks

File: README.md [hunk 2]
```diff
@@ -7,23 +7,61 @@
...
+
+Below are the instructions to verify the binary using `cosign` for version `v1.1.0`.
+
+- Use an env variable for the version
+
+```shell
+export KARMOR_VERSION="1.1.0"
+```
+
+- Download release tarball, certificate, and signature files
```
Comment (kranurag7) [inline README.md:36]:
```suggestion
- Download released tarball, certificate, and signature files
```

**Review by kranurag7 — CHANGES_REQUESTED** [review]
@rod4n4m1 some final nits. 

Thank you, the changes looks good to me. 

Also please sign-off and squash the commits.

File: README.md [hunk 3]
```diff
@@ -7,23 +7,61 @@
...
+curl -LO https://github.com/kubearmor/kubearmor-client/releases/download/v${KARMOR_VERSION}/karmor_${KARMOR_VERSION}_linux_amd64.tar.gz
+
+curl -LO https://github.com/kubearmor/kubearmor-client/releases/download/v${KARMOR_VERSION}/karmor_${KARMOR_VERSION}_linux_amd64.tar.gz.cert
 
+curl -LO https://github.com/kubearmor/kubearmor-client/releases/download/v${KARMOR_VERSION}/karmor_${KARMOR_VERSION}_linux_amd64.tar.gz.sig
 ```
+
+</details>
+
+- Verify the release tarball integrity with `cosign`
```
[the remark this answers is not in the document]
Comment (rod4n4m1) [reply README.md:51]: Done.

File: README.md [hunk 4]
```diff
@@ -7,23 +7,61 @@
...
+
+<details>
+  <summary>Details</summary>
```
[the remark this answers is not in the document]
Comment (rod4n4m1) [reply README.md:54]: Done

File: README.md [hunk 5]
```diff
@@ -7,23 +7,61 @@
...
+- Use an env variable for the version
+
+```shell
+export KARMOR_VERSION="1.1.0"
+```
+
+- Download release tarball, certificate, and signature files
+
+<details>
+  <summary>Details</summary>
```
[the remark this answers is not in the document]
Comment (rod4n4m1) [reply README.md:39]: Done.

File: README.md (hunk 2 above)
Comment (rod4n4m1) [reply README.md:36]: Done.

File: README.md [hunk 6]
```diff
@@ -7,23 +7,61 @@
...
 ```
 
-## Usage
+### Steps to Verify the Binary (Recommended)
+
+We sign all releases with `cosign`, therefore we recommend verifying **karmor** tarball prior to its installation.
+
+Below are the instructions to verify the binary using `cosign` for version `v1.1.0`.
+
+- Use an env variable for the version
```
[the remark this answers is not in the document]
Comment (rod4n4m1) [reply README.md:30]: Done.

**Comment by rod4n4m1** [comment]
@kranurag7 I think I made it. Please check.

**Review by kranurag7 — APPROVED** [review]
Thank you @rod4n4m1 for the patch and addressing the reviews in between. 🥇

**Pull request merged**